examples/geckocamera_example.cpp

```diff
@@ -60,10 +62,10 @@ class GeckoCameraExample : CameraListener
                 }
 
                 shared_ptr<Camera> camera;
-                if (manager->openCamera(cameraList.at(0).id, camera)) {
+                if (manager->openCamera(cameraList.at(cameraNumber).id, camera)) {
                     camera->setListener(this);
-                    if (camera->startCapture(caps.at(0))) {
-                        this_thread::sleep_for(chrono::seconds(10));
+                    if (camera->startCapture(caps.at(modeNumber))) {
```

Comment:
Where do you check that modeNumber stays within valid bounds?

Reply:
Fixed, thank you.